velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMain.java

```diff
@@ -0,0 +1,57 @@
+package me.leoko.advancedban.velocity;
+
+import com.google.inject.Inject;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.event.proxy.ProxyShutdownEvent;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.velocity.listener.ChatListenerVelocity;
+import me.leoko.advancedban.velocity.listener.ConnectionListenerVelocity;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+@Plugin(
+        id = VelocityMain.VelocityPluginInfo.PLUGIN_ID,
+        name = VelocityMain.VelocityPluginInfo.PLUGIN_NAME,
+        version = "${project.version}",
+        authors = {"Leoko", "Fede1132(Added velocity support)"}
+)
+public class VelocityMain {
+    public static class VelocityPluginInfo {
+        public static final String PLUGIN_ID = "advancedban";
```

Comment:
Why are this class and its constants public?

Reply:
As I said the idea at begininng was different.. so I forgot to update it